UltiSnips/snippets.snippets

```diff
@@ -2,8 +2,8 @@ priority -50
 
 # We use a little hack so that the snippet is expanded
 # and parsed correctly
-snippet snip "Snippet definition" b
-`!p snip.rv = "snippet"` ${1:Tab_trigger} "${2:Description}" ${3:b}
+snippet snip "Snippet definition" bs
```

Comment:
that is too generic. It strips whitespaces from all lines - but there might be significant whitespaces in the contents of the snippet at the end. You have to special case the first line.